src/open_inwoner/cms/cases/tests/test_htmx.py

```diff
@@ -43,6 +45,9 @@
 
 @tag("e2e")
 @requests_mock.Mocker()
+@patch.object(
+    ContactmomentenClient, "retrieve_objectcontactmomenten_for_zaak", autospec=True
+)
```

Comment:
See my other note. Why is this here and not a request mock?

Reply:
I had a mock earlier but it wasn't actually used (only returned an empty list), so I switched to `patch` per Steven's request.

Reply:
Can you add a `return_value=[]` so it is explicit we want this to return an empty list?